tests/testcases/mod.rs

```diff
@@ -321,7 +327,23 @@ pub fn bootstrap_smoke_test(
     };
     let mut build = std::process::Command::new(spin_binary());
     // Ensure `spin` is on the path
-    build.env("PATH", path).args(["build"]);
+    build.env("PATH", &path).args(["build"]);
+    build_env_vars.iter().for_each(|(key, value)| {
+        if key == "PATH" {
+            let mut custom_path = value.to_owned();
+            if value.starts_with('.') {
+                let current_dir = env.path();
+                custom_path = current_dir
+                    .join(value)
+                    .to_str()
+                    .unwrap_or_default()
+                    .to_owned();
+            }
+            build.env(key, format!("{}:{}", custom_path, path));
```

Comment:
I think this needs to be `"{};{}"` on Windows.

Reply:
The existing expression above your changes (`format!("{path}:{}", spin_binary().parent().unwrap().display())`) should probably also be changed.  Or is this code never intended to run on Windows?

Reply:
I think getting the integration tests to be cross-platform is something on the backlog. @rylev would be the best person for that question.